### PR TITLE
doc: Convert doc/README.md to doc/README.adoc

### PR DESCRIPTION
**Pull request by jpereira**

### CODE REVIEW

File: doc/.gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
 _build
 *.html
-*.adoc
```
Comment (alandekok) [inline doc/.gitignore]: The point is that we *don't* want to accidentally commit the auto-generated adoc files.

Perhaps by adding a `doc/raddb/.gitignore` file, with appropriate content.

Comment (jpereira) [reply doc/.gitignore]: @alandekok It's ready into the commit. I tested and didn't add any auto-generated *.adoc files.

image

**Comment by arr2036** [comment]
This needs updating for the current contents of doc/README.md

**Comment by jpereira** [comment]
ready updated.

**Pull request merged**